sql/patch_89_90_d.sql

```diff
@@ -0,0 +1,27 @@
+-- Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+-- Copyright [2016-2017] EMBL-European Bioinformatics Institute
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+# patch_89_90_d.sql
+#
+# Title: Add species_id field
+#
+# Description:
+# Add species_id column in the mapping_session table is a possible solution to help identifying the strain with the multiSpecies database mapping sessions.
+
+ALTER TABLE mapping_session ADD COLUMN species_id INT(10) UNSIGNED NOT NULL DEFAULT 1 ,ADD KEY ( species_id );
+										  							  
+# Patch identifier
+
+INSERT INTO meta (species_id, meta_key, meta_value) VALUES (NULL, 'patch', 'patch_89_90_d.sql|add_species_id');	
```

Comment:
this should be patch_90_91_b.sql, the changes will not be ready for this release